specs/APIReview_AllowDrop.md

```diff
@@ -0,0 +1,98 @@
+# Background
+Currently the drag&drop functionality is default enabled in webview2 and there is no way to disable it. Some developers may want to disbale this functionality in their applications 
+based upon their scenarios. According to such feature requirements, we add the new API to provide developers with the capability to configure the drag&drop functionality.
+
+# Description
+We add a new `AllowDrop` property in `CoreWebView2Controller`. This API allows end developers to toggle the drag&drop functionality easily.
+If it's disabled, any drag&drop actions will keep out of work. By default, it's disabled to accommodate to conventions.
```

Comment:
Usually we want the default value for a new API to be consistent with the behavior we had before the API so that we don't introduce a compat problem. For instance in this case if we add AllowDrop that is disabled by default but previously it was enabled then when the WebView2 Runtime updates to the newer version with AllowDrop disabled  on an end user's machine an app that may have depended upon drag and drop working will now be broken.

Maybe this is OK if we know that there aren't apps depending on this right now?

Reply:
Very make sense. I'm not sure if there are some existing apps that are already depending on the default drag&drop functionality. The reason I made it default false before just be that I think that may be more reasonable for most new applications. But from the perspective of consistency and compatibility, keeping default to true would definitely be more reasonable. I made the changes in both this spec and code. Thanks.